mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -659,3 +668,31 @@ func testAccNetappVolume_volumeBasicExample_cleanupScheduledBackup(t *testing.T,
 		return nil
 	}
 }
+
+func testAccNetappVolume_volumeBasicExample_autoTier(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_netapp_storage_pool" "default" {
```

Comment:
note: the actual failure reason is because we're recreating the `google_netapp_storage_pool` with both the delete and create in parallel. That's happening because in this config the resource is named (in Terraform) `google_netapp_storage_pool.default2` before and `google_netapp_storage_pool.default` now, so Terraform doesn't recognise it should wait for the deletion before creating the new resource.

We should obviate that with the other proposed changes.

Reply:
Done